src/test/java/com/google/finapp/FinAppTests.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2020 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.finapp;
+
+import com.google.cloud.ByteArray;
+import com.google.cloud.spanner.DatabaseClient;
+import com.google.cloud.spanner.DatabaseId;
+import com.google.cloud.spanner.Key;
+import com.google.cloud.spanner.KeySet;
+import com.google.cloud.spanner.ResultSet;
+import com.google.cloud.spanner.Spanner;
+import com.google.cloud.spanner.SpannerOptions;
+import java.math.BigDecimal;
+import java.util.Arrays;
+import java.util.UUID;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FinAppTests {
+
+  private static SpannerDaoInterface JDBCDao;
+  private static SpannerDaoInterface JavaDao;
+  private static DatabaseClient databaseClient;
+
+  @BeforeClass
+  public static void setUpTests() {
+    // set all IDs according to emulator setup
+    String spannerProjectId = "test-project";
+    String spannerInstanceId = "test-instance";
+    String spannerDatabaseId = "test-database";
+    JDBCDao = new SpannerDaoJDBCImpl(spannerProjectId, spannerInstanceId, spannerDatabaseId);
+    SpannerOptions spannerOptions = SpannerOptions.getDefaultInstance();
+    Spanner spanner = spannerOptions.toBuilder().build().getService();
+    databaseClient = spanner
+        .getDatabaseClient(DatabaseId.of(spannerProjectId, spannerInstanceId, spannerDatabaseId));
+    JavaDao = new SpannerDaoImpl(databaseClient);
+  }
+
+  @Test
+  public void createAccountTest() throws SpannerDaoException {
+    ByteArray accountIdJDBC = UuidConverter.getBytesFromUuid(UUID.randomUUID());
+    ByteArray accountIdJava = UuidConverter.getBytesFromUuid(UUID.randomUUID());
```

Comment:
I use UUIDs for the sake of convenience, but this can be changed. 

UUIDs make it so that the database doesn't need to be cleared between each tests. If we had some set IDs, then running the tests twice in a row could cause the second tests to pass if the first did--even if something broke the code--since the test would query the info placed in the database prior.

One other alternative is to clear the database in the setup (and after the tests). This would make it so that we can have set database IDs. However, clearing the database might be undesirable for the user--although the test data we put in would technically "corrupt" the data in the database anyways.